superbench/config/default.yaml

```diff
@@ -46,9 +46,10 @@ superbench:
     nccl-bw:default:
       enable: true
       modes:
-        - name: local
+        - name: mpi
```

Comment:
why change this? it should be n node do local nccl test previously,  'all-nodes' with proc_num=1 is meant for all n-nodes mpi or mpi for each node

Reply:
if for test, I think you should add a new one such as 'nccl-bw:allnode' in default.yaml, rather than change original one

Reply:
Yeah, I think adding new sections is the right way in this case, will update later. Thanks for the comment.